File: app/src/main/java/com/example/pcmall/model/Address.java
package com.example.pcmall.model;

import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

@NoArgsConstructor
@AllArgsConstructor
@Data
@Accessors(chain = true)
public class Address {
    private Integer id;//地址编号
    private String uid;//用户编号
    private String province;//省
    private String city;//市
    private String district;//区
    private String address;//详细地址
    private String receiverName;//收件人
    private String phone;//手机号码
    private LocalDateTime createTime;//创建时间
    private LocalDateTime updateTime;//修改时间
    private Integer isDefault;//是否选中（0不选中 1选中）
    private Integer isDelete;//是否删除（）
}

File: core/model/src/main/java/com/example/pcmallcompose/core/model/Address.kt
package com.example.pcmallcompose.core.model

import com.fasterxml.jackson.annotation.JsonFormat
import com.fasterxml.jackson.databind.annotation.JsonDeserialize
import com.fasterxml.jackson.databind.annotation.JsonSerialize
import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer
import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer
import java.time.LocalDateTime

data class Address(
    val id: Int, //地址编号
    val uid: String, //用户编号
    val province: String, //省
    val city: String, //市
    val district: String, //区
    val addressDetail: String, //详细地址
    val receiverName: String, //收件人
    val phone: String, //手机号码
    @field:JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    @field:JsonDeserialize(using = LocalDateTimeDeserializer::class)
    @field:JsonSerialize(using = LocalDateTimeSerializer::class)
    val createTime: LocalDateTime, //创建时间
    @field:JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    @field:JsonDeserialize(using = LocalDateTimeDeserializer::class)
    @field:JsonSerialize(using = LocalDateTimeSerializer::class)
    val updateTime: LocalDateTime? = null, //修改时间
    val isDefault: Int, //是否选中（0不选中 1选中）
)